src/tlo/methods/depression.py

```diff
@@ -580,7 +585,10 @@ def do_when_suspected_depression(
         manager and run diagnosis tests.
         - If the diagnosis_function is passed in directly, it is assumed to be a Callable method that
         runs diagnosis tests.
+<<<<<<< HEAD
+=======
 
+>>>>>>> master
```

Comment:
```suggestion
```

Note to self to remove this before hitting merge... but don't want to restart the tests!